Makefile

```diff
@@ -51,7 +53,7 @@ integration_test:
 smoketest:
 	$(MAKE) DSS_TEST_MODE="integration" tests/test_smoketest.py
 
-scaletest:
+scaletest: mypy lint
```

Comment:
I don't think scaletest should depend on linter targets

Reply:
Done.